stage/testing/ubuntu/noble/package-model.json

```diff
@@ -1,5 +1,4 @@
 {
```

Comment:
looks like this file can be removed altogether as there are no more overrides

Reply:
That's right. But I specifically didn't remove it for visibility and history.